### PR TITLE
#66 Project List Card Styling

### PR DESCRIPTION
**Pull request by brittanygay**
Connect #66

- stubbed out the Cards for the Project List (Discover) page
- Added placeholders for now, we can either hide them or keep them up if we plan on hooking everything up 

screencapture-localhost-4000-styleguide-2018-08-08-13_35_34

screen shot 2018-08-08 at 1 20 37 pm
screen shot 2018-08-08 at 1 34 55 pm

### CODE REVIEW

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.prod.revelry.net/relint/revelrylabs/incentivize/100).

File: lib/incentivize_web/templates/repository/index.html.eex [hunk 1]
```diff
@@ -2,42 +2,67 @@
...
-                <div class="rev-Col">
-                  <h3>
-                    <a href="<%= repository_path(@conn, :show, repo.owner, repo.name) %>">
-                      <%= repo.owner %>/<%= repo.name %>
-                    </a>
-                  </h3>
+          <div class="rev-Col rev-Col--medium6">
+            <div class="rev-Card undefined rev-CardLayout">
+
+              <div class="rev-Card-header undefined rev-CardLayout-bar">
```
Comment (jwietelmann) [inline lib/incentivize_web/templates/repository/index.html.eex]: lol why `undefined` everywhere

Comment (brittanygay) [reply lib/incentivize_web/templates/repository/index.html.eex]: Womp that's weird. It was set up like that originally, not sure why those were there. I will remove!

File: assets/css/utilities/_typography.scss [hunk 2]
```diff
@@ -7,3 +7,6 @@
 .u-noPadding-top {
   padding-top: 0;
 }
+.u-center {
```
Comment (blazebarsamian) [inline assets/css/utilities/_typography.scss]: You can delete this and use the Harmonium styles for `Text-center`

**Comment by brittanygay** [comment]
@blazebarsamian just pushed up edits for the above comments!

**Pull request merged**